server/chart_config/climate.json

```diff
@@ -54,5 +54,18 @@
     ],
     "unit": "celsius",
     "isOverview": false
+  },
+  {
+    "category": "Climate",
+    "titleId": "CHART_TITLE-Mean_concentration_PM2.5",
+    "title": "Mean concentration of PM 2.5",
```

Comment:
" Mean PM2.5 concentration" seems to be the more common way to describe this.

Reply:
Done